### ISSUE TITLE
/manage_preorder

### ISSUE THREAD

**Issue by Jonassala** [Verkami-Staff/TEST#3]
Se reaprovechan casi todas las cajas del Settings y 1 del Dashboard actuales.

# Esqueleto /manage_preorder

- Caja: Pasos + Notificar a verkami
- Caja: Portada, vídeo e información básica 
- Caja: Descripción del proyecto
- Caja: Recompensas
- Caja: Formularios 
- Caja: Datos bancarios y de Facturación
- Caja: PayPal
- Caja gris: Ayuda Te puede ser útil saber…

# Diseño

## Pasos + Notificar a Verkami

![Captura de Pantalla 2019-09-13 a les 14 08 58](https://user-images.githubusercontent.com/2071003/64861368-0e684700-d630-11e9-992b-abb44c494b9b.png)

Caja superior con el **color característico** que se asigne al nuevo estado preorder.

Aprovechar la plantilla de la caja azul de 3 pasos que aparece en los proyectos Approved.
En ella se explicará **cómo funciona la “Preventa”** junto a un botón de **“Notificar a Verkami”**.

- Al pinchar botón **Notificar a Verkami** que se repase si **se han creado todos los Formularios** y se hayan introducido todos los **Campos de Facturación**. 
Que se muestre un modal genérico con el texto de: `Faltan rellenar alguno de los campos obligatorios: Datos bancarios y de facturación o crear los Formularios para los mecenas.`

- [ ] Podríamos revisarlo desde el staff si están todos los campos si complica?

- Con la **preventa en marcha**, el botón de **Notificar a Verkami** que pase a ser: **Parar preventa** que al pincharlo abra un modal explicando si quiere parar la preventa con confirmación y enviar un mailer a admins vk para que lo lleven a cabo. 

# Funcionalidades específicas

## Cerrar Preventa

El autor puede notificar a Verkami que quiere cerrar la preventa

**Comment by aitor** [comment]
> Analytics y Píxel de Facebook

¿Vamos a permitir trackers distintos para la preventa a los que haya definidos en el proyecto?? (en el principio desde un punto de vista de analítica no tiene mucho sentido porque una persona revisando datos quiere sobre todo evolución y con esto empezariamos de cero).

**Comment by Jonassala** [comment]
Cierto, no estaba eliminado en la captura de la caja correspondiente. Utilizamos los mismos trackers para todo, y si los quieren modificar, que lo hagan desde /manage_settings.

- Eliminamos pues los trackers de la página de /manage_preorder. Editado en la definición https://github.com/Verkami-Staff/PREORDER-APORTACION/issues/3#issue-493300017

**Comment by Jonassala** [comment]
# Diseños

![Notify verkami(1)](https://user-images.githubusercontent.com/2071003/80070617-83471c00-8543-11ea-8171-c9f5b6b66ef4.png)

- Falta cómo se sube y se gestionan las imágenes fullscreen y las disntitas aplicaciones: Cards y Responsive.

**Comment by aitor** [comment]
Varios puntos sobre el diseño:

* En aquellas secciones que siguen aparentemente la misma estructura visual que otras que ya tenemos (eg. esa sección negra para publicar) surge la disyuntiva de si intentar adaptar los templates existente al nuevo contexto para que podamos reutilizarlos tal cual o si crear nuevos que sean muy similares para ser utilizados en la preventa. 

  Esto lo iremos respondiendo caso a caso, pero en general la postura será de duplicar, porque aunque suponga un pequeño coste inicial y coste continuo posterior de mantenimiento nos da independencia para evolucionar ambos ambitos sin restriccciones. Obviamente habrá casos en los que los cambios sean tan minimos o estes razonablmentes seguros de que queremos conservarlos en linea que nos digan lo contrario pero como principio lo que he comentado.

* En la gestión de rewards falta el tema de la imagen. Imagino que es un tema de relleno pero bueno por concretar.

**Comment by Jonassala** [comment]
> Esto lo iremos respondiendo caso a caso, pero en general la postura será de duplicar, porque aunque suponga un pequeño coste inicial y coste continuo posterior de mantenimiento nos da independencia para evolucionar ambos ambitos sin restriccciones. Obviamente habrá casos en los que los cambios sean tan minimos o estes razonablmentes seguros de que queremos conservarlos en linea que nos digan lo contrario pero como principio lo que he comentado.

Ok!

> En la gestión de rewards falta el tema de la imagen. Imagino que es un tema de relleno pero bueno por concretar.

Sips, en esta captura era para mostrar la cabecera, los apartados a continuación no son los que tocan ni en el orden que toca (que es el definido en este ticket).
